spec/report_spec.rb

```diff
@@ -1267,4 +1267,16 @@ def gloops
       }
     end
   end
+
+  it 'should include device data when notify is called' do
```

Comment:
I prefer avoiding "should" in doc strings (https://github.com/rubocop-hq/rspec-style-guide#should-in-example-docstrings) to avoid the feeling of repeating "should" in most doc strings.